delfin/task_manager/scheduler/schedule_manager.py

```diff
@@ -77,6 +77,9 @@ def schedule_boot_jobs(self):
                 LOG.error("Failed to initialize periodic tasks, reason: %s.",
                           six.text_type(e))
                 raise e
+        # Recover the job in db
```

Comment:
I think self.scheduler.add_job actions done above this, will be removed as it is no more needed.  recover_job will do setting/resetting executors.

Reply:
Yes, the `add_job` part would be removed after all the content of distributor.__call__ have been removed.